dtmi/com/microchip/sam_iot_wm-1.json

```diff
@@ -0,0 +1,239 @@
+{
+  "@context": "dtmi:dtdl:context;2",
+  "@id": "dtmi:com:Microchip:SAM_IoT_WM;1",
+  "@type": "Interface",
+  "displayName": "SAM-IoT WM",
+  "description": "Reports device temperature, light intensity, and the current state of the 2 buttons & 4 LEDs.  Provides ability to turn on/off any of the 4 LEDs.",
+  "contents": [
+    {
+      "@type": [
+        "Telemetry",
+        "Temperature"
+      ],
+      "name": "temperature",
+      "displayName": {
+        "en": "Temperature"
+      },
+      "description": {
+        "en": "Temperature in degrees Celsius from Microchip MCP9808 high-accuracy temperature sensor"
+      },
+      "schema": "integer",
+      "unit": "degreeCelsius"
+    },
+    {
+      "@type": [
+        "Telemetry",
+        "Illuminance"
+      ],
+      "name": "light",
+      "displayName": {
+        "en": "Brightness from light sensor"
+      },
+      "description": {
+        "en": "Brightness in illuminance from Vishay TEMT6000X01 ambient light sensor"
+      },
+      "schema": "integer",
+      "unit": "lux"
+    },
+    {
+      "@type": [
+        "Telemetry"
+      ],
+      "name": "button_event",
+      "displayName": {
+        "en": "SW0/SW1 button push event"
+      },
+      "description": {
+        "en": "Event triggered when button is pressed"
+      },
+      "schema": {
+        "@type": "Object",
+        "fields": [
+          {
+            "name": "button_name",
+            "schema": "string"
+          },
+          {
+            "name": "press_count",
+            "schema": "integer"
+          }
+        ]
+      }
+    },
+    {
+      "@type": [
+        "Property"
+      ],
+      "name": "led_b",
+      "displayName": {
+        "en": "Blue LED state"
+      },
+      "description": {
+        "en": "Returns current state of the Blue LED. If True, the Blue LED is on and the WiFi AP is connected."
+      },
+      "schema": "dtmi:com:Microchip:SAM_IoT_WM:LedState;1",
+      "writable": false
+    },
+    {
+      "@type": [
+        "Property"
+      ],
+      "name": "led_g",
+      "displayName": {
+        "en": "Green LED state"
+      },
+      "description": {
+        "en": "The current state of the Green LED. When it is on, SAM IoT is connected to Cloud."
+      },
+      "schema": "dtmi:com:Microchip:SAM_IoT_WM:LedState;1",
+      "writable": false
+    },
+    {
+      "@type": [
+        "Property"
+      ],
+      "name": "led_r",
+      "displayName": {
+        "en": "Red LED state"
+      },
+      "description": {
+        "en": "The current state of the Red LED. When it is on or blinking, SAM IoT experienced error(s)."
+      },
+      "schema": {
```

Comment:
Any reason to do not reuse the same schema as `led_g` or `led_y` properties?

Reply:
You are correct - there is no reason that the **led_r** property cannot use the same schema as the **led_b**, **led_g**, & **led_y** properties.  I have pushed the updated JSON file so please review the changes and merge into the main branch.  Thank you!